tests/unit/test_noise_form.py

```diff
@@ -160,7 +175,17 @@ def test_noise_order(mocker, dummy_data, dataset):
     # function is called. Here we grab the string of the noise type for one mock method
     # call and not the second method.
     call_order = [x[0] for x in mock.mock_calls if type(x[1][0]) == str]
-    row_order = [row_noise_type.name for row_noise_type in dataset.row_noise_types]
+    row_order = [
```

Comment:
Why is this change required?

Reply:
This test is pretty confusing at this point but to answer your question - not all datasets have guardian based duplication now (only census) but we parametrize the test to get a confg for each dataset with every noise type to make sure they will run it in order. No noise is actually happening but we do create a config with all noise types. Earlier in this test we mock the return of the noise functions like this instead of getting a list of the datasets noise types.